src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -407,7 +407,7 @@
     <value>Change '{0}' to be read-only by removing the property setter</value>
   </data>
   <data name="OverloadOperatorEqualsOnOverridingValueTypeEqualsTitle" xml:space="preserve">
-    <value>Overload operator equals on overriding value type Equals</value>
+    <value>A value type overrides System.Object.Equals but does not implement the equality operator</value>
```

Comment:
Use single quotes around types.

```suggestion
    <value>A value type overrides 'System.Object.Equals' but does not implement the equality operator</value>
```

We tend to keep short sentences for title but I am not sure how to shrink your message.

Reply:
@Evangelink I took it just a copy/paste.

Things that can make it a (little) bit shorter:

- "the equality operator" -> "equality operators" (a little bit shorter, and more accurate to be plural I think (to represent == and !=)
- "does not" -> "doesn't" (but I'm not sure if you follow a specific convention for this and want to be consistent)
- "System.Object.Equals" -> "object.Equals" (Note lower case object to make it still accurate, but shorter at the same time)

I'll wait for @mavasani to weigh in for this.